gcs-action/LICENSE

```diff
@@ -0,0 +1,201 @@
+Apache License
```

Comment:
IANAL, but I believe the top-level license suffices. @craigdbarber thoughts?

Reply:
@sethvargo is correct, no need to duplicate the LICENSE for each sub-folder.

Reply:
Done